imap_processing/glows/glows_cdf_attrs.py

```diff
@@ -0,0 +1,190 @@
+
+text = (
+    "The GLObal Solar Wind Structure (GLOWS) is a non-imaging single-pixel Lyman-alpha "
+    "photometer that will be used to observe the sky distribution of the helioglow to "
+    "better understand the evolution of the solar wind structure. "
+    "The Lyman-alpha photon counts from these observations can be used to build a more "
+    "comprehensive picture of the solar wind structure and how it changes through the "
+    "solar cycles. "
+    "GLOWS design and assembly is led by the Space Research Center, Warsaw, Poland "
+    "(CBK PAN). See https://imap.princeton.edu/instruments/glows for more details."
+)
+
+glows_base = GlobalInstrumentAttrs(
+    __version__,
+    "GLOWS>GLObal Solar Wind Structure",
+    text,
+    "Photons (space)",  # TODO: is this accurate?
+)
+
+# TODO: Do we want a data format version as well as a software version?
+
+glows_l1a_hist_attrs = GlobalDataLevelAttrs(
+    "L1A_hist>Level-1A histogram",
+    logical_source="imap_glows_l1a_hist",
+    logical_source_desc="IMAP Mission GLOWS Histogram Level-1A Data.",
+    instrument_base=glows_base,
+)
+
+glows_l1a_de_attrs = GlobalDataLevelAttrs(
+    "L1A_de>Level-1A direct event",
+    logical_source="imap_glows_l1a_de",
+    logical_source_desc="IMAP Mission GLOWS Direct Event Level-1A Data.",
+    instrument_base=glows_base,
+)
+
+bins_attrs = AttrBase(
+    validmin=0,
+    validmax=70,  # doc says maximum count per bin is 66.7
+    catdesc="Counts of direct events for photon impacts.",
+    fieldname="Counts of direct events.",
+    format="F7.4",  # Float with 4 digits
+    var_type="support_data",
+    display_type="time_series",
+    label_axis="Counts",
+)
+
+# TODO Update this
+per_second_attrs = AttrBase(
+    validmin=0,
+    validmax=300,
+    catdesc="Direct events recorded approximately per second",
+    fieldname="List of direct events ",
+    format="F7.4",  # Float with 4 digits
+    var_type="support_data",
+    display_type="time_series",
+    label_axis="Counts",
+)
+
+# TODO Update this
+# TODO rename this
+direct_event_attrs = AttrBase(
+    validmin=0,
+    validmax=300,  # TODO what is a reasonable max
+    catdesc="Direct events recorded approximately per second",
+    fieldname="List of direct events ",
+    format="F7.4",  # Int with 10 digits
+    var_type="data",
+    display_type="time_series",
+    label_axis="Counts",
+)
+
+histogram_attrs = ScienceAttrs(
+    validmin=0,
+    validmax=70,  # doc says maximum count per bin is 66.7, value has a 71
+    catdesc="Histogram of photon counts.",
+    depend_0="epoch",
+    depend_1="bins",
+    fieldname="Histogram of photon counts.",
+    format="F7.4",  # Float with 4 digits TODO: are histogram objects float or int?
+    display_type="time_series",
+    label_axis="Counts",
+    fill_val=GlobalConstants.INT_FILLVAL,
+    units="counts",
+    var_type="data",
+)
+
+metadata_attrs = ScienceAttrs(
+    depend_0="epoch",
+    validmin=GlobalConstants.INT_FILLVAL,
```

Comment:
Andriy’s seems to not like fillval as min value. He wants us to use some min value instead. May be zero?

Reply:
Good call, I will investigate a good value for this. Unfortunately it's used for a bunch of attributes but they're decently similar